rollup/Makefile

```diff
@@ -7,10 +7,17 @@ mock_abi:
 	cd .. && solc --evm-version cancun --bin --abi --optimize --overwrite -o ./build/bin ./rollup/mock_bridge/MockBridge.sol
 	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --bin=./build/bin/MockBridge.bin --abi=./build/bin/MockBridge.abi --pkg=mock_bridge --out=./rollup/mock_bridge/MockBridge.go
 
+libzstd:
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+	@sudo mv libzktrie.so /usr/local/lib
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
+	@sudo mv libscroll_zstd.so /usr/local/lib
+
 rollup_bins: ## Builds the Rollup bins.
-	go build -o $(PWD)/build/bin/event_watcher ./cmd/event_watcher/
-	go build -o $(PWD)/build/bin/gas_oracle ./cmd/gas_oracle/
-	go build -o $(PWD)/build/bin/rollup_relayer ./cmd/rollup_relayer/
+	$(MAKE) libzstd
```

Comment:
can delete this line

Reply:
refactored in https://github.com/scroll-tech/scroll/pull/1298/commits/7d5f90bfe523c0bdd1bc90cd7255c98df6a76827.